target/linux/ath79/patches-5.4/940-ath79-add-qcn550x.patch

```diff
@@ -0,0 +1,35 @@
+--- a/arch/mips/ath79/early_printk.c
++++ b/arch/mips/ath79/early_printk.c
```

Comment:
Is this only needed for 5.4? If not, please add a patch for 5.10 as well.

Please also add a header to the patch, so it looks like after using "git format-patch" when submitted upstream ...

Reply:
Done